docs/source/plotting/index.rst

```diff
@@ -0,0 +1,563 @@
+   ax_bottom.tick_params(axis='both', direction='in')
+   ax_bottom.set_ylabel('Difference')
+   fig.tight_layout()
+   #fig.show()
+   
+
+====================
+Plotting Sample Logs
+====================
+
+The :func:`mantid.plots.MantidAxes.plot<mantid.plots.MantidAxes.plot>` function can show sample logs. By default,
+the time axis represents the time since the first proton charge pulse (the
+beginning of the run), but one can also plot absolute time using `FullTime=True`
+
+.. plot::
+   :include-source:
+   
+   import matplotlib.pyplot as plt
+   from mantid import plots
+   from mantid.simpleapi import Load
+   
+   w=Load('CNCS_7860')
+   fig = plt.figure()
+   ax1 = fig.add_subplot(211,projection='mantid')
+   ax2 = fig.add_subplot(212,projection='mantid')
+   ax1.plot(w, LogName='ChopperStatus5')
+   ax1.set_title('From run start')
+   ax2.plot(w, LogName='ChopperStatus5',FullTime=True)
+   ax2.set_title('Absolute time')
+   fig.tight_layout()
+   #fig.show()
+   
+
+Note that the parasite axes in matplotlib do not accept the projection keyword.
+So one needs to use :func:`mantid.plots.plotfunctions.plot<mantid.plots.plotfunctions.plot>` instead.
+
+.. plot::
+   :include-source:
+   
+   import matplotlib.pyplot as plt
+   from mantid import plots
+   from mantid.simpleapi import Load
+   w=Load('CNCS_7860')
+   fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
+   ax.plot(w,LogName='ChopperStatus5')
+   axt=ax.twiny()
+   plots.plotfunctions.plot(axt,w,LogName='ChopperStatus5', FullTime=True)
+   #fig.show()
+
+
+=============
+Complex plots
+=============
+
+One common type of a slightly more complex figure involves drawing an inset.
+
+.. plot::
+   :include-source:
+
+   import matplotlib.pyplot as plt
+   import numpy as np
+   from mantid import plots
+   from mantid.simpleapi import CreateWorkspace, FFT
+   from matplotlib import rcParams
+   import warnings
+
+   x=np.linspace(0,10,250)
+   y=np.cos(2*np.pi*1.1*x)*np.exp(-x/7.)
+   err=np.sqrt(0.01+x/200.)
+   w=CreateWorkspace(x,y,err,UnitX='tof')
+   fft=FFT(w)
+
+   # make all ticks point in
+   rcParams['xtick.direction'] = 'in'
+   rcParams['ytick.direction'] = 'in'
+
+   fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
+   ax.errorbar(w,'ks')
+   ax.set_ylabel('Asymmetry')
+   ax.set_ylim(-1.5,2)
+   ax_inset=fig.add_axes([0.7,0.72,0.2,0.2],projection='mantid')
+   ax_inset.plot(fft,specNum=6)
+   ax_inset.set_xlim(0,2)
+   ax_inset.set_xlabel('Frequency (MHz)')
+   ax_inset.set_ylabel('|FFT|')
+   # note that thight_layout will produce a warning here "This figure includes
+   # Axes that are not compatible with tight_layout, so its results might be incorrect."
+   with warnings.catch_warnings():
+       warnings.simplefilter("ignore", category=UserWarning)
+       fig.tight_layout()
+   #fig.show()
+
+
+Plotting dispersion curves  on multiple panels can also be done using matplotlib:
+
+.. plot::
```

Comment:
I only get the final segment when I do the last plotting example:
![image](https://user-images.githubusercontent.com/13132128/36667114-c04aaf34-1ae4-11e8-85a9-1c70acdb38b6.png)